src/mlpack/methods/kde/kde_main.cpp

```diff
@@ -292,9 +290,9 @@ void BINDING_FUNCTION(util::Params& params, util::Timers& timers)
 
     // Set Mode.
     if (modeStr == "dual-tree")
-      kde->Mode() = KDEMode::DUAL_TREE_MODE;
+      kde->Mode() = KDEMode::KDE_DUAL_TREE_MODE;
     else if (modeStr == "single-tree")
-      kde->Mode() = KDEMode::SINGLE_TREE_MODE;
+      kde->Mode() = KDEMode::KDE_SINGLE_TREE_MODE;
   }
```

Comment:
Yeah---the `DUAL_TREE_MODE` name is already taken by the `NSMode` enum (which is used for various neighbor searching classes).  This might not be the best possible solution, but I think it'll work for now.